contracts/utility-token/UtilityToken.sol

```diff
@@ -51,6 +49,50 @@ contract UtilityToken is ERC20Token {
 
     /* External Functions */
 
+    /**
```

Comment:
Please, define `\pre` and `\post` documentation tags for this external function.

Reply:
Done